### PR TITLE
readme.md

### PR DESCRIPTION
**Pull request by veryjohn1991**
Added readme file as per our group discussion based ideas and requirements .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h1>Food Picker 3000</h1>
             <p>Don't know what to eat? We got you!</p>
           </div>
     </header>
-
+<!-- checking it -->
```
Comment (Soliske) [inline index.html:16]: Don't forget to delete this

File: README.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+- The application should have a feature to rate and provide feedback on the delivered food and overall restaurant experience.
+
+- It should have a responsive customer support system to address any queries or issues related to ordering food or watching a movie simultaneously.
+=======
 # Food Picker 3000
 
 Sources:
 Emojis:
 Slot Machine animation:
+
```
Comment (Soliske) [inline README.md:38]: Awesome job on the PR. However, with this app we want it to pick a food item near the user, NOT order it for them. They do that on their own outside of the app.

**Review by Soliske — CHANGES_REQUESTED** [review]
Update the user story to match the comments left in the review.

File: README.md (hunk 2 above)
Comment (veryjohn1991) [reply README.md:38]: ohh ok i will modify  it

**Pull request merged**